changelog-2022.07.27-3.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/qt/res/app.organicmaps.desktop.metainfo.xml b/qt/res/app.organicmaps.desktop.metainfo.xml
+index 9174f31..4495b30 100644
+--- a/qt/res/app.organicmaps.desktop.metainfo.xml
++++ b/qt/res/app.organicmaps.desktop.metainfo.xml
+@@ -107,6 +107,19 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2022.07.27-3" date="2022-07-27">
++     <description>
++       <ul>
++         <li>Fixed invalid operations on Wayland for OpenGL ES3 capable devices</li>
++         <li>Use OpenGL ES3 by default on Linux</li>
++         <li>New OSM data as of July 18</li>
```

Comment:
Here is the full relevant list of desktop changes in addition to opengl ones:

• New OSM data as of July 18, 2022
• Added missing shop types from OpenStreetMap tags: antiques, art, baby_goods, bag, boutique, charity, cheese, craft, dairy, deli, electrical, farm, fishing, grocery, health_food, houseware, interior_decoration, kitchen, lottery, medical_supply, nutrition_supplements, paint, pastry, perfumery, second_hand, sewing, storage_rental, tobacco, trade, watches, wholesale
• Properly select max speed in some routing cases
• sport=* tags have higher priority than leisure=pitch
• Fill transit mark place page with feature info
• Show text info about the next turn's street longer
• Use building:min_level for 3D buildings
• Allow removing cuisines from selection in the Editor
• Fix Belarusian, German, Russian, and Ukrainian translations
• Add missing translations for OSM natural=* and amenity=events_venue types

Reply:
We'll publish it at https://organicmaps.app/news in case you need to double-check.

Reply:
Thanks, I took over what is on https://organicmaps.app/news